package/tests/helpers.py

```diff
@@ -1,6 +1,47 @@
-
 class Any(object):
     def __init__(self, predicate=None):
         self.predicate = predicate
     def __eq__(self, other):
         return not self.predicate or self.predicate(other)
+
+def mocked_requests_get(*args, **kwargs):
+    '''
+    Helper method for mocking requests for test_script_downloader
+    '''
+    repo_dict = {
+        "public": 'https://raw.repocontentservice.com/SomeUser/SomePublicRepo/master/bashScript.sh', 
+        "private_token": 'https://raw.repocontentservice.com/SomeUser/SomePrivateTokenRepo/master/bashScript.sh',
+        "private_cred": 'https://raw.repocontentservice.com/SomeUser/SomePrivateCredRepo/master/bashScript.sh',
+        "content": 'SomeBashScriptContent'
+    }
+
+    class MockResponse:
+        def __init__(self, json_data, status_code, headers, url):
+            self.json_data = json_data
+            self.status_code = status_code
+            self.headers = headers
+            self.url = url
+        
+        def json(self):
+            return self.json_data
+
+        def iter_content(self, chunk):
+            return self.json_data
+
+    if args[0] == repo_dict['public']:
```

Comment:
python 3.10 will have switch statements :D

Reply:
well, I'll be sure to change it when we upgrade 3 years from now...